### PR TITLE
Add Willow destination to Segment Doc

### PR DESCRIPTION
**Pull request by sneha-shashidhar**
### Proposed changes

Included the Segment documentation for Willow, a destination integration submitted via Dev Center. 

### Merge timing

- ASAP once approved

### Related issues (optional)

None

### CODE REVIEW

**Comment by markzegarelli** [comment]
Hi @sneha-shashidhar, the doc you uploaded is empty. Do you have the content for this doc?

**Comment by sneha-shashidhar** [comment]
Hi @markzegarelli Apologies. The content should be visible now. Can you please check?

File: src/connections/destinations/catalog/willow/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: Willow Destination
+rewrite: true
+id: 620ebe78b4e75580b6e6b72a
+---
+
+[Willow](https://heywillow.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a customer support platform built for early stage startups. We focus on getting your whole team (even engineering) solving issues together with fun commenting/tagging, showing your everything in one place from from customer messages to in-app actions, and showing your entire customer's journey in one continuous feed from day 1 to today.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:7]:
```suggestion
[Willow](https://heywillow.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a customer support platform for early stage startups. It focuses on getting your whole team (even engineering) to solve issues together with commenting and tagging, shows you everything in one place from customer messages to in-app actions, and it shows your entire customer's journey in one continuous feed from day one to today.
```

Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:7]:
```suggestion
[Willow](https://heywillow.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a customer support platform for early stage startups. It focuses on getting your whole team (even engineering) to solve issues together with commenting and tagging, shows you everything in one place from customer messages to in-app actions, and it shows your entire customer's journey in one continuous feed from day one to today.
```

File: src/connections/destinations/catalog/willow/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Getting Started
+
+{% include content/connection-modes.md %}
+
+1. From the Destinations catalog page in the Segment App, click **Add Destination**.
+2. Search for "Willow" in the Destinations Catalog, and select the "Willow" destination.
+3. Choose which Source should send data to the "Willow" destination.
+4. Select the correct team from your [Willow workspace](https://heywillow.io/a/workspace){:target="\_blank"}.
+5. Go to your team's settings page, select "API Keys", then find and copy the "API key".
+6. Enter the "API Key" in the "Willow" destination settings in Segment.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:20]:
```suggestion
2. Search for **Willow** in the Destinations Catalog, and select the **Willow** destination.
3. Choose which Source should send data to the Willow destination.
4. Select the team from your [Willow workspace](https://heywillow.io/a/workspace){:target="\_blank"}.
5. Go to your team's settings page and select **API Keys**, and copy the **API key**.
6. Enter the **API Key** in the Willow destination settings in Segment.
```

Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:20]:
```suggestion
2. Search for **Willow** in the Destinations Catalog, and select the **Willow** destination.
3. Choose which Source should send data to the Willow destination.
4. Select the team from your [Willow workspace](https://heywillow.io/a/workspace){:target="\_blank"}.
5. Go to your team's settings page and select **API Keys**, and copy the **API key**.
6. Enter the **API Key** in the Willow destination settings in Segment.
```

File: src/connections/destinations/catalog/willow/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## Supported methods
+
+Willow supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:23]:
```suggestion
Willow supports the following methods as specified in the [Segment Spec](/docs/connections/spec).
```
```suggestion
Willow supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
```

File: src/connections/destinations/catalog/willow/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+```obj-c
+[[SEGAnalytics sharedAnalytics] screen:@"Home"];
+```
+
+Segment sends Screen calls to Willow as a `Viewed Page`.
+
+### Identify
+
+Send [Identify](/docs/connections/spec/identify) calls to Willow to add traits to the customer. These are viewable in the lifetime view and customer overview. For example:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:47]:
```suggestion
Send [Identify](/docs/connections/spec/identify) calls to Willow to add traits to the customer. You can view these in the lifetime view and customer overview. For example:
```

Comment (stayseesong) [inline src/connections/destinations/catalog/willow/index.md:47]:
```suggestion
Send [Identify](/docs/connections/spec/identify) calls to Willow to add traits to the customer. You can view these in the lifetime view and customer overview. For example:
```

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**